scripts/data_ingestion/ingest_current_season.py

```
@@ -0,0 +1,220 @@
+        description="Orchestrate current MLB season data ingestion"
+    )
+    parser.add_argument("--season", type=int, help="Season to ingest (e.g., 2026)")
+    parser.add_argument("--days-back", type=int, help="Number of days back to ingest (default: 7)")
+    parser.add_argument("--schedule-only", action="store_true", help="Only fetch schedule, don't ingest games")
+    parser.add_argument("--skip-bridge", action="store_true", help="Skip bridge table population")
+    parser.add_argument("--skip-xref", action="store_true", help="Skip game cross-reference completion")
+    parser.add_argument("--no-validate", action="store_true", help="Skip validation step")
+    
+    args = parser.parse_args()
+    
+    season = args.season or datetime.now().year
```

Ruff (0.15.10)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)